$NetBSD: patch-aa,v 1.4 2022/01/22 18:50:23 pho Exp $

refuse(3) used to not support -ho in option_list. Already fixed in
HEAD.

--- src/wdfs-main.c.orig	2007-04-12 08:30:08.000000000 +0000
+++ src/wdfs-main.c
@@ -137,8 +137,14 @@ static int wdfs_opt_proc(
 	switch (key) {
 		case KEY_HELP:
 			print_help();
+#if !defined(__NetBSD__) || FUSE_H_ >= 20211204
+/*
+ * XXX Using refuse(3), this resulted in abnormal program termination
+ *     with SIG_SEGV.
+ */
 			fuse_opt_add_arg(option_list, "-ho");
 			call_fuse_main(option_list);
+#endif
 			exit(1);
 
 		case KEY_VERSION:
